Standards/scs-0003-v1-sovereign-cloud-standards-yaml.md

```diff
@@ -10,19 +10,47 @@ track: Global
 The [Sovereign Cloud Stack (SCS)](https://scs.community) provides standards for a range of cloud infrastructure types.
 It strives for interoperable and sovereign cloud offerings which can be deployed and used by a wide range of organizations and individuals.
 
-With a growing number of documents and certification levels for Sovereign Cloud Stack, it is becoming increasingly difficult to keep track
-of all stable, obsoleted as well as all upcoming standards. This decision record describes how we use technically describe
-certification levels and the mandatory standards therein.
+SCS plans to offer six types of certificates, which are sorted into two dimensions:
+
+1. _certification level_, of which there are three:
+   - SCS-compatible
+   - SCS-open
+   - SCS-sovereign
+2. _cloud layer_, of which there are two:
+   - infastructure as a service (IaaS)
+   - Kubernetes as a service (KaaS)
```

Comment:
I think we need a decision on how to call this layer and implement it everywhere. The team is currently called `Team Container` and often CaaS is preferred over KaaS, as it doesn't only hold Kubernetes.

Reply:
Yes, let's talk about that in the SIG meeting on 2023-08-24. As of right now, `kaas` is the term used in the yaml and this document, and I stuck to it.